app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
```

Comment:
- [ ] The views in devise for the registration and login should be modified to match the wireframes and their styling. Kindly do so.

Your design
![image](https://user-images.githubusercontent.com/81932478/216176898-7ce69b96-893f-42b5-8a65-9ce091d5fa9e.png)

Given Wireframe
![image](https://user-images.githubusercontent.com/81932478/216177020-5b334387-58d9-4ba2-ade7-8b51cf82a61b.png)